ejb/WFLY-13828-remote+tls-is-not-supported-by-EJBClient-and-remote-outbound-connection.adoc

```diff
@@ -0,0 +1,49 @@
+= remote+tls is not supported by EJBClient and remote-outbound-connection
+:email:             tomasz.adamski@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+=== Issue
+* https://issues.redhat.com/browse/WFLY-13828
+
+=== Dev Contacts
+
+* mailto:tadamski@redhat.com
+
+=== QE Contacts
+
+=== Testing By
+[x] Engineering
+
+[] QE
+
+=== Affected Projects or Components
+EJB
+Remoting
+
+== Introduction
+Customers would like to be able to configure EJB to use remote+tls protocol. This feature is already implemented in remoting but the configuration has to be added to jboss-ejb-client library and WildFly in order to make it work.
+
+== Implementation
+
+After the change users would be able to configure remote+tls protocol in remoting subsystem's remote-outbound-connection:
+
+[source]
+----
+<subsystem xmlns="urn:jboss:domain:remoting:4.1">
+   <http-connector name="http-remoting-connector" connector-ref="default" security-realm="ApplicationRealm"/>
```

Comment:
I know this RFE is more about being able to specify the protocol but as this is a TLS configuration I think it would be good to also have an example referencing Elytron capabilities as the legacy security realm will be deleted at some point in the upcoming WildFly releases.